src/Microsoft.DocAsCode.Build.Engine/LinkPhaseHandler.cs

```diff
@@ -221,14 +224,14 @@ private InternalManifestItem GetManifestItem(FileModel model, SaveResult result)
             };
         }
 
-        private static void Postbuild(HostService hostService)
+        private static void Postbuild(HostService hostService, AggregatedPerformanceScope aggregatedPerformanceScope)
         {
             BuildPhaseUtility.RunBuildSteps(
                 hostService.Processor.BuildSteps,
                 buildStep =>
                 {
                     Logger.LogVerbose($"Processor {hostService.Processor.Name}, step {buildStep.Name}: Postbuilding...");
-                    using (new LoggerPhaseScope(buildStep.Name, LogLevel.Verbose))
+                    using (new LoggerPhaseScope(buildStep.Name, LogLevel.Verbose, aggregatedPerformanceScope))
```

Comment:
>, aggregatedPerformanceScope [](start = 80, length = 28)

For PostBuild, this LoggerPhaseScope already contains all the files. You can see the prameter below is `hostService.Models`. We do not need aggregate here.

Same for PreBuild.